packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -327,18 +332,18 @@ export class AtomicSearchInterface {
   }
 
   private initUrlManager() {
-    if (!this.reflectStateInUrl) {
-      return;
-    }
```

Comment:
We now always initialize the url manager.

I could make it so we update facet dependencies on a different listener (e.g., breadcrumb manager) when `reflectStateInUrl` is `false`, but I prefer to consistently use the same listener regardless of the use case.